### PR TITLE
Add typings for atom-keymap library

### PR DESCRIPTION
**Pull request by enlight**

### CODE REVIEW

**Comment by vvakame** [comment]
*atom-keymap/atom-keymap.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/atom-keymap
  * http://bower.io/search/?q=atom-keymap
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (atom-keymap/atom-keymap-tests.ts or others)
* [ ] pass the Travic-CI test?

File: atom-keymap/atom-keymap.d.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
+// Type definitions for first-mate v5.1.5
+// Project: https://github.com/atom/first-mate/
```
Comment (vvakame) [inline atom-keymap/atom-keymap.d.ts]: is this right? https://github.com/atom/atom-keymap ?

**Comment by enlight** [comment]
Good catch, I've amended the previous commit to fix the project name & URL and force pushed.

**Comment by vvakame** [comment]
@enlight :+1:

**Pull request merged**